commit 60891b08b52be73555c0fac766167e8f0768d622
Author: Mark Rutland <mark.rutland@arm.com>
Date:   Thu Feb 2 17:32:14 2017 +0000

    arm64: fix erroneous __raw_read_system_reg() cases
    
    commit 7d0928f18bf890d2853281f59aba0dd5a46b34f9 upstream.
    
    Since it was introduced in commit da8d02d19ffdd201 ("arm64/capabilities:
    Make use of system wide safe value"), __raw_read_system_reg() has
    erroneously mapped some sysreg IDs to other registers.
    
    For the fields in ID_ISAR5_EL1, our local feature detection will be
    erroneous. We may spuriously detect that a feature is uniformly
    supported, or may fail to detect when it actually is, meaning some
    compat hwcaps may be erroneous (or not enforced upon hotplug).
    
    This patch corrects the erroneous entries.
    
    Signed-off-by: Mark Rutland <mark.rutland@arm.com>
    Fixes: da8d02d19ffdd201 ("arm64/capabilities: Make use of system wide safe value")
    Reported-by: Catalin Marinas <catalin.marinas@arm.com>
    Cc: Suzuki K Poulose <suzuki.poulose@arm.com>
    Cc: Will Deacon <will.deacon@arm.com>
    Signed-off-by: Will Deacon <will.deacon@arm.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1300_60891b08b52be73555c0fac766167e8f0768d622.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1300_60891b08b52be73555c0fac766167e8f0768d622.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
